app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list) -> int:
```

Comment:
Indicate that you expect to obtain a list of `Player` class instances for `players` in the type annotation